libraries/AC_Fence/AC_PolyFence_loader.h

```diff
@@ -251,42 +258,55 @@ class AC_PolyFence_loader
     // can be found:
     Vector2f *_loaded_return_point;
 
+    // pointer into _loaded_coordinates where the return point
+    // can be found:
+    Vector2l *_loaded_return_point_lla;
+
     class InclusionBoundary {
     public:
         Vector2f *points; // pointer into the _loaded_offsets_from_origin array
+        Vector2l *lla_points; // pointer into the _loaded_coordinates array
         uint8_t count; // count of points in the boundary
     };
     InclusionBoundary *_loaded_inclusion_boundary;
+
     uint8_t _num_loaded_inclusion_boundaries;
 
     class ExclusionBoundary {
     public:
         Vector2f *points; // pointer into the _loaded_offsets_from_origin array
+        Vector2l *lla_points; // pointer into the _loaded_coordinates array
         uint8_t count; // count of points in the boundary
     };
     ExclusionBoundary *_loaded_exclusion_boundary;
+
     uint8_t _num_loaded_exclusion_boundaries;
 
     // _loaded_offsets_from_origin - stores x/y offset-from-origin
     // coordinate pairs.  Various items store their locations in this
     // allocation - the polygon boundaries and the return point, for
     // example.
     Vector2f *_loaded_offsets_from_origin;
+    Vector2l *_loaded_coordinates;
 
     class ExclusionCircle {
     public:
-        Vector2f pos_cm;
+        Vector2f pos_cm;    // vector offset from home in cm
+        Vector2l loc;       // lat/lng of zone
```

Comment:
normally variables named "loc" are Locations.. again, not a big deal. just noticing the difference.

Reply:
Ack - updated